controllers/users.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const { encode } = require('../utils');
+let db = require('../services/storage'),
+  encrypt = require('../services/encrypt'),
+  bus;
+
+/**
+ * Adds an user into the db and publishes the action to the bus
+ * @param {object} data
+ * @returns {Promise}
+ */
+function createUser(data = {}) {
+  let { username, password, provider, auth } = data,
```

Comment:
Looks like these can be `const` too 😅 
```suggestion
  const { username, password, provider, auth } = data,
```